exporter/tanzuobservabilityexporter/README.md

```diff
@@ -1,21 +1,29 @@
 # Tanzu Observability (Wavefront) Exporter
 
-This exporter supports sending traces to [Tanzu Observability](https://tanzu.vmware.com/observability).
+This exporter supports sending both traces and metrics to [Tanzu Observability](https://tanzu.vmware.com/observability).
 
 ## Prerequisites
 
 - [Obtain the Tanzu Observability by Wavefront API token.](https://docs.wavefront.com/wavefront_api.html#generating-an-api-token)
 - [Set up and start a Tanzu Observability by Wavefront proxy](https://docs.wavefront.com/proxies_installing.html) and configure it with the API token you obtained.
 - To have the proxy generate [span RED metrics](https://docs.wavefront.com/trace_data_details.html#red-metrics) from trace data, [configure](https://docs.wavefront.com/proxies_configuring.html) the proxy's `customTracingListenerPorts` and use it for the exporter's endpoint.
 
-## Data Conversion
+## Data Conversion for Traces
 
 - Trace IDs and Span IDs are converted to UUIDs. For example, span IDs are left-padded with zeros to fit the correct size.
 - Events are converted to [Span Logs](https://docs.wavefront.com/trace_data_details.html#span-logs).
 - Kind is converted to the `span.kind` tag.
 - Status is converted to `error`, `status.code` and `status.message` tags.
 - TraceState is converted to the `w3c.tracestate` tag.
 
+## Data Conversion for Metrics
+- Cumulative sum metrics are converted to counter metrics
+- Delta sum metrics are converted to delta counters
+- Each cumulative histogram metric is converted to multiple counter metrics (one counter metric per bucket in the histogram). Each counter metric has a special "le" tag that matches the upper bound of the corresponding bucket. The count in the counter metric is the sum of the corresponding bucket and all the buckets before it.
+- Delta histograms get converted to histograms in wavefront
+- Exponential histograms are converted the same way as regular histograms
+- Each summary metric is converted to multiple gauge metrics (one gauge metric for each quantile in the summary).  A special "quantile" tag contains a value between 0 and 1 indicating the quantile for which the value belongs.
+
 ## Tanzu Observability Specific Attributes
 
```

Comment:
Let's replace this and the two sub-bullets with:

- Tanzu Observability Spans require two [Application Tags](https://docs.wavefront.com/trace_data_details.html#how-wavefront-uses-application-tags): `application` and `service`. If missing, the exporter defaults them to `application=defaultApp` and `service=defaultService`.

Reply:
Actually, my comment doesn't account for `service.name`. We'll pair up to figure out the wording.